src/VM/OpcodeHandler.h

```diff
@@ -27,6 +27,9 @@
 
 // Third party includes
 
+// Marks handler parameter as unused and prevents compiler warnings.
+#define UNUSED_ARG(arg) (void)(arg)
```

Comment:
Not exactly, the code 
/* int movie = */ _vm->dataStack()->popInteger();
can lead to warning about unused function value, IIRC

It can be something like
/* int movie = */ (void)_vm->dataStack()->popInteger();

is it better?